spec/controllers/student_identifier_types_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe StudentIdentifierTypesController, type: :controller do
+  include ActiveJob::TestHelper
+
+  let(:organization)  { GitHubFactory.create_owner_classroom_org }
+  let(:user)          { organization.users.first                 }
+  let(:student)       { GitHubFactory.create_classroom_student   }
+
+  let(:student_identifier_type) { GitHubFactory.create_student_identifier(organization) }
+
+  before do
+    sign_in(user)
+    Classroom.flipper[:student_identifier].enable
+  end
```

Comment:
Can we add a small set of tests, to ensure that we are returning a `404` if the flag is not on?